docs/kyma/04-03-cluster-installation.md

```diff
@@ -0,0 +1,294 @@
+
+  </details>
+  
+  <details>
+  <summary label="AKS">
+  AKS
+  </summary>
+
+- [Microsoft Azure](https://azure.microsoft.com) account
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+>**NOTE:** Running Kyma on AKS requires three [`Standard_D4_v3` machines](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-1). You create these machines when you complete the **Prepare the cluster** step.
+
+  </details>
+
+  <details>
+  <summary label="IBM Cloud">
+  IBM Cloud
+  </summary>
+
+- [IBM Cloud](https://cloud.ibm.com/login) account
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+- [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started)
+
+  </details>
+
+  <details>
+  <summary label="Gardener">
+  Gardener
+  </summary>
+
+- [Gardener](https://gardener.cloud/) seed cluster
+- [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled or a [Microsoft Azure](https://azure.microsoft.com) account
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+
+  </details>
+
+</div>
+
+## Choose the release to install
+
+1. Go to [this](https://github.com/kyma-project/kyma/releases/) page and choose the release you want to install.
+
+2. Export the release version as an environment variable. Run:
+
+    ```bash
+    export KYMA_VERSION={KYMA_RELEASE_VERSION}
+    ```
+
+## Prepare the cluster
+
+<div tabs name="prepare-cluster" group="cluster-installation">
+  <details>
+  <summary label="GKE">
+  GKE
+  </summary>
+  
+1. Select a name for your cluster. Export the cluster name, the name of your GCP project, and the [zone](https://cloud.google.com/compute/docs/regions-zones/) you want to deploy to as environment variables. Run:
+
+    ```bash
+    export CLUSTER_NAME={CLUSTER_NAME_YOU_WANT}
+    export GCP_PROJECT={YOUR_GCP_PROJECT}
+    export GCP_ZONE={GCP_ZONE_TO_DEPLOY_TO}
+    ```
+
+2. Create a cluster in the defined zone. Run:
+
+    ```bash
+    gcloud container --project "$GCP_PROJECT" clusters \
+    create "$CLUSTER_NAME" --zone "$GCP_ZONE" \
+    --cluster-version "1.14" --machine-type "n1-standard-4" \
+    --addons HorizontalPodAutoscaling,HttpLoadBalancing
+    ```
+
+3. Configure kubectl to use your new cluster. Run:
+
+    ```bash
+    gcloud container clusters get-credentials $CLUSTER_NAME --zone $GCP_ZONE --project $GCP_PROJECT
+    ```
+
+4. Add your account as the cluster administrator:
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value account)
+    ```
+  
+  </details>
+
+  <details>
```

Comment:
To be consistent with the  **Prerequisites** section, I would add the IBM Cloud tab in between the AKS and Gardener (line 195).